src/stack-hci/azext_stack_hci/aaz/latest/stack_hci/cluster/_update.py

```diff
@@ -59,20 +59,42 @@ def _build_arguments_schema(cls, *args, **kwargs):
         # define Arg Group "Cluster"
 
         _args_schema = cls._args_schema
+        _args_schema.identity = AAZObjectArg(
+            options=["--identity"],
+            arg_group="Cluster",
+            help="Identity of Cluster resource",
+            nullable=True,
+        )
         _args_schema.tags = AAZDictArg(
             options=["--tags"],
             arg_group="Cluster",
             help="Resource tags.",
             nullable=True,
         )
 
+        identity = cls._args_schema.identity
+        identity.type = AAZStrArg(
+            options=["type"],
+            help="Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).",
+            enum={"None": "None", "SystemAssigned": "SystemAssigned", "SystemAssigned, UserAssigned": "SystemAssigned, UserAssigned", "UserAssigned": "UserAssigned"},
+        )
+        identity.user_assigned_identities = AAZDictArg(
+            options=["user-assigned-identities"],
+            help="The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
+            nullable=True,
+        )
+
+        user_assigned_identities = cls._args_schema.identity.user_assigned_identities
+        user_assigned_identities.Element = AAZObjectArg(
+            nullable=True,
+            blank={},
```

Comment:
Same as above. Please note this is a update related scenario, we need to follow this [guideline](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md#operate-managed-identity-on-existing-resource) to create a `az <resource> identity assign` sub-command.